hubspot3/test/test_tickets.py

```diff
@@ -116,38 +116,26 @@ def test_get_all_tickets(tickets_client, mock_connection):
     assert set(third_ticket["properties"].keys()) == {"prop_2"}
 
 
-@pytest.mark.parametrize(
-    "extra_properties_given, extra_properties_as_list",
-    [(None, []),
-     ("lead_source", ["lead_source"]),
-     (["hs_analytics_last_url", "hs_analytics_revenue"],
-      ["hs_analytics_last_url", "hs_analytics_revenue"]
-      )
-     ]
-)
-def test_get_batch(
-    tickets_client,
-    mock_connection,
-    extra_properties_given,
-    extra_properties_as_list,
-):
+def test_get_batch(tickets_client, mock_connection):
     response_body = {
         "3234574": {"objectId": 3234574, "properties": {}},
         "3234575": {"objectId": 3234575, "properties": {}},
     }
     ids = ["3234574", "3234575"]
-    properties = TicketsClient.default_batch_properties.copy()
-    properties.extend(extra_properties_as_list)
+
+    # This extra properties are enough to generate 2 requests
+    extra_properties = [str(num) for num in range(2000)]
 
     mock_connection.set_response(200, json.dumps(response_body))
```

Comment:
If the code is doing 2 requests it should be getting 2 responses, if you keep de ids and put different parameters you can test your join at the end, like this:

`response_body_1 = {
        "3234574": {"objectId": 3234574, "properties": {
"a": {                    "value": "1571427728000",
                    "timestamp": 1571411169418,
                    "source": "CALCULATED",
                    "sourceId": "TicketsRollupProperties"}
}},
        "3234575": {"objectId": 3234575, "properties": {"a": {                    "value": "1571427728000",
                    "timestamp": 1571411169418,
                    "source": "CALCULATED",
                    "sourceId": "TicketsRollupProperties"}}},
    }`


`response_body_2 = {
        "3234574": {"objectId": 3234574, "properties": {"b": {                    "value": "1571427728000",
                    "timestamp": 1571411169418,
                    "source": "CALCULATED",
                    "sourceId": "TicketsRollupProperties"}}},
        "3234575": {"objectId": 3234575, "properties": {"b": {                    "value": "1571427728000",
                    "timestamp": 1571411169418,
                    "source": "CALCULATED",
                    "sourceId": "TicketsRollupProperties"}}},
    }`

`mock_connection.set_response([(200, json.dumps(response_body_1)), (200, json.dumps(response_body_2))])`

Reply:
If you like this could became a future card